### ISSUE TITLE
No files found in App

### ISSUE THREAD

**Issue by martin-77** [nextcloud/android#8783]
**Steps to reproduce**

    Log into Nextcloud with the app
    click All Files
    No files appear

**Expected behaviour**

    You should see your files an folders

**Actual behaviour**

    the app tells me that there are no folders, BUT: activities are fine, if i click on a file there, the file is shown and can be opened...
    removed files are shown too, and Server-messages like calender-events are shown inside the App
    strange about this: this only happens to LDAP - Users. local users are able to use the App without any problems.

Can you reproduce this problem on https://try.nextcloud.com?

no, web is fine
Environment data

Android version: 11

Device model: does not make a difference / Samsung S12 / RealMe7pro / 

Stock or customized system: Stock

Nextcloud app version: 3.17.0 RC2

Nextcloud server version:22

Reverse proxy: Synology

Maybe related to: https://github.com/nextcloud/android/issues/7660

### Logs

```
2021-07-29T07:40:40.418+0200;I;RefreshFolderOperation;Got display name: com.owncloud.android.lib.common.UserInfo@d1856b8
2021-07-29T07:40:40.418+0200;D;RefreshFolderOperation;Checking changes in username@domain.tld@orga.domain.tld/
2021-07-29T07:40:40.460+0200;I;RefreshFolderOperation;Got display name: com.owncloud.android.lib.common.UserInfo@582664
2021-07-29T07:40:40.460+0200;D;RefreshFolderOperation;Checking changes in username@domain.tld@orga.domain.tld/
2021-07-29T07:40:40.477+0200;D;OwnCloudClient #0;REQUEST PROPFIND /remote.php/webdav/
2021-07-29T07:40:40.481+0200;D;OwnCloudClient #0;REQUEST PROPFIND /remote.php/webdav/
2021-07-29T07:40:40.611+0200;I;RefreshFolderOperation;Checked username@domain.tld@orga.domain.tld/ : changed
2021-07-29T07:40:40.612+0200;D;OwnCloudClient #0;REQUEST PROPFIND /remote.php/webdav/
2021-07-29T07:40:40.627+0200;I;RefreshFolderOperation;Checked username@domain.tld@orga.domain.tld/ : changed
2021-07-29T07:40:40.628+0200;D;OwnCloudClient #0;REQUEST PROPFIND /remote.php/webdav/
2021-07-29T07:40:40.750+0200;E;ReadFolderRemoteOperation;Synchronized /: The Nextcloud server is not configured!
2021-07-29T07:40:40.750+0200;D;RefreshFolderOperation;Synchronizing username@domain.tld@orga.domain.tld/
2021-07-29T07:40:40.766+0200;E;ReadFolderRemoteOperation;Synchronized /: The Nextcloud server is not configured!
2021-07-29T07:40:40.766+0200;D;RefreshFolderOperation;Synchronizing username@domain.tld@orga.domain.tld/
2021-07-29T07:40:40.767+0200;D;RefreshFolderOperation;Send broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_CONTENTS_SYNCED
2021-07-29T07:40:40.772+0200;D;FileDisplayActivity;Received broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_CONTENTS_SYNCED
2021-07-29T07:40:40.772+0200;D;RefreshFolderOperation;Send broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_SHARES_SYNCED
2021-07-29T07:40:40.797+0200;D;RefreshFolderOperation;Send broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_CONTENTS_SYNCED
2021-07-29T07:40:40.798+0200;D;RefreshFolderOperation;Send broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_SHARES_SYNCED
2021-07-29T07:40:40.830+0200;D;FileDisplayActivity;Setting progress visibility to true
2021-07-29T07:40:40.830+0200;D;FileDisplayActivity;Received broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_SHARES_SYNCED
2021-07-29T07:40:40.878+0200;D;FileDisplayActivity;Setting progress visibility to false
2021-07-29T07:40:40.879+0200;D;FileDisplayActivity;Received broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_CONTENTS_SYNCED
2021-07-29T07:40:40.928+0200;D;FileDisplayActivity;Setting progress visibility to true
2021-07-29T07:40:40.928+0200;D;FileDisplayActivity;Received broadcast com.owncloud.android.operations.RefreshFolderOperation.EVENT_SINGLE_FOLDER_SHARES_SYNCED
2021-07-29T07:40:40.979+0200;D;FileDisplayActivity;Setting progress visibility to false
2021-07-29T07:40:58.548+0200;V;FileDisplayActivity;onPause() start
2021-07-29T07:40:58.579+0200;D;FileDisplayActivity;onPause() ending
2021-07-29T07:40:58.580+0200;V;FileDisplayActivity;onPause() end
2021-07-29T07:40:58.606+0200;D;SettingsActivity;onCreate(Bundle) starting
2021-07-29T07:40:58.919+0200;D;SettingsActivity;onStart() starting
2021-07-29T07:40:58.939+0200;D;SettingsActivity;onResume() starting
2021-07-29T07:40:59.466+0200;D;FileDisplayActivity;onStop() ending
2021-07-29T07:40:59.468+0200;V;FileDisplayActivity;onSaveInstanceState() start
2021-07-29T07:40:59.469+0200;D;FileDisplayActivity;onSaveInstanceState(Bundle) starting
2021-07-29T07:40:59.470+0200;D;ExtendedListFragment;onSaveInstanceState()
2021-07-29T07:40:59.470+0200;V;FileDisplayActivity;onSaveInstanceState() end
2021-07-29T07:41:06.370+0200;D;SettingsActivity;onPause() ending
2021-07-29T07:41:06.389+0200;D;LogsActivity;onCreate(Bundle) starting
```

**Comment by martin-77** [comment]
Bug seems to be resolved in the latest beta 22.1.0 RC1

**Comment by martin-77** [comment]
22.1.0 is installed no more problems. 
Closed
